src/app/systelab-components/styles/sass/_signature-canvas.scss

```diff
@@ -0,0 +1,23 @@
+systelab-signature-canvas {
+    & > div {
+        padding:15px;
+    }
+    canvas{
+        border: 1px solid #d3d3d3;
+    }
+    & .alert{
+        padding: 0.5rem 0.3rem;
+    }
+    & .optionsBar{
+        margin-top:10px;
+    }
+    & .slab-form{
+        padding:15px;
+    }
+    & .mt-15{
```

Comment:
NEVER DO THIS!!!!!!!

Reply:
What is not advisable? use this class key or to refer to the child items that have that class?